lib/seahorn/BvOpSem2.cc

```diff
@@ -3242,12 +3292,12 @@ void Bv2OpSem::initCrabAnalysis(const llvm::Module &M) {
 void Bv2OpSem::runCrabAnalysis() {
   /// Set Crab parameters
   clam::AnalysisParams aparams;
-  aparams.dom = clam::CrabDomain::INTERVALS;
+  aparams.dom = CrabDom;
   aparams.run_inter = true;
-  aparams.check = clam::CheckerKind::NOCHECKS;
+  aparams.check = clam::CheckerKind::ASSERTION;
```

Comment:
I don't think we need the `ASSERTION` option. This will run the checker in Crab but we don't need/want that